apps/portal/src/app/connect/pay/overview/page.mdx

```diff
@@ -74,30 +75,21 @@ Universal Bridge is supported on select EVM compatible chains. To view the full
 | Stripe   | https://docs.stripe.com/crypto/onramp                                                                   |
-
-Specify a preferred onramp provider by setting the preferredProvider parameter in ConnectButton, PayEmbed, or Headless Integration. For details, refer to 
-[customization](../pay/customization/connectbutton#preferredProvider) or the [API Reference for preferredProvider](/references/typescript/v5/GetBuyWithFiatQuoteParams).
+| Coinbase | https://docs.cdp.coinbase.com/onramp/docs/payment-methods/ |
 
 ## Integration Options
 
 | OPTION                                                                                              | BEST FOR                                                                                                                                  |
 | --------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
-| [ConnectButton](/connect/pay/get-started#option-1-connectbutton)                                    | Developers who want out-of-the-box support complete with wallet and social login.                                                         |
-| [PayEmbed](/connect/pay/get-started#option-2-embed-pay)                                             | Developers who want onramp and crypto purchase experiences directly in their application.                                                 |
-| [Transaction Flow (sendTransaction)](/connect/pay/get-started#option-3-send-a-transaction-with-pay) | Developers who want users to onramp or purchase crypto directly into a transaction. Great for minting and NFT purchase flows.             |
-| [Custom Universal Bridge Experience](/connect/pay/build-a-custom-experience)                                     | Developers who want full customization over every step of the onramp process, from displaying quotes to polling for transaction statuses. |
+| [ConnectButton](https://playground.thirdweb.com/connect/sign-in/button)                             | Developers who want out-of-the-box support complete with wallet and social login.                                                         |
+| [PayEmbed](https://playground.thirdweb.com/connect/pay/fund-wallet)                                 | Developers who want onramp and crypto purchase experiences directly in their application.                                                 |
+| [Transaction Flow](https://playground.thirdweb.com/connect/pay/transactions) | Developers who want users to onramp or purchase crypto directly into a transaction. Great for minting and NFT purchase flows.             |
+| [Custom Universal Bridge Experience](https://playground.thirdweb.com/connect/pay/transactions)      | Developers who want full customization over every step of the onramp process, from displaying quotes to polling for transaction statuses. |
```

Comment:
This link and above is the same?

Reply:
there were two flows on the same page- I thought that second one was the custom flow - the Automatic Onramp one

Reply:
which one should custom link to? thought it was this
![CleanShot 2025-03-26 at 22 33 33@2x](https://github.com/user-attachments/assets/87e67fd2-6d9d-4f03-a16e-ebbffc8435c3)

Reply:
I would just do

Buy crypto - /pay
In app purchase - /commerce
Transactions - /transactions 
Headless - /backend